controller/handler/trial.js

```diff
@@ -121,6 +121,11 @@ function trialView (request, reply) {
                     patient.totalMissed = 0;
                 }
 
+                patient.dateStarted = moment(patient.dateStarted)
+                    .format('MM-DD-YYYY');
```

Comment:
Yes I see that on patient's view we are using format 'MM-DD'YYYY'
Then do not change it.
Let this be.
LGTM